Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -81,18 +86,48 @@ namespace O3DE::ProjectManager
         connect(m_gemInspector, &GemInspector::UninstallGem, this, &GemCatalogScreen::UninstallGem);
 
         QWidget* filterWidget = new QWidget(this);
-        filterWidget->setFixedWidth(240);
+        filterWidget->setFixedWidth(sidePanelWidth);
         m_filterWidgetLayout = new QVBoxLayout();
         m_filterWidgetLayout->setMargin(0);
         m_filterWidgetLayout->setSpacing(0);
         filterWidget->setLayout(m_filterWidgetLayout);
 
-        GemListHeaderWidget* listHeaderWidget = new GemListHeaderWidget(m_proxyModel);
+        GemListHeaderWidget* catalogHeaderWidget = new GemListHeaderWidget(m_proxyModel);
+
+        const int headerTableMinWidth = MinWindowWidth - sidePanelWidth * 2 - GemItemDelegate::s_itemMargins.left() -
+            GemItemDelegate::s_itemMargins.right() - verticalScrollBarWidth;
+
+        constexpr int minHeaderSectionWidth = 100;
+        AdjustableHeaderWidget* listHeaderWidget = new AdjustableHeaderWidget(
+            QStringList{ tr("Gem Name"), tr("Gem Summary"), tr("Status") },
+            QVector<int>{
+                GemItemDelegate::s_defaultSummaryStartX - 30,
+                headerTableMinWidth - GemItemDelegate::s_defaultSummaryStartX - GemItemDelegate::s_buttonWidth - GemItemDelegate::s_extraSummarySpacing - GemItemDelegate::s_itemMargins.left() - GemItemDelegate::s_contentMargins.right(),
```

Comment:
I wonder if we need all this math now that this header section is set to "stretch"